br/pkg/lightning/checkpoints/checkpoints.go

```diff
@@ -1615,10 +1616,16 @@ func (cpdb *MySQLCheckpointsDB) IgnoreErrorCheckpoint(ctx context.Context, table
 		Logger: log.FromContext(ctx).With(zap.String("table", tableName)),
 	}
 	err := s.Transact(ctx, "ignore error checkpoints", func(c context.Context, tx *sql.Tx) error {
+		if strings.ContainsRune(cpdb.schema, ';') || strings.ContainsRune(CheckpointTableNameEngine, ';') {
```

Comment:
By tracking up how this varaible is set, you can see we have used `common.EscapeIdentifier` to escape the value, so I guess no SQL injection will happen. We don't need to check `;` here

@bb7133 PTAL

Reply:
Get it, I think we do not need further check here